Workbooks/Azure Private MEC/PMEC Site Overview/PMEC Site Overview.workbook

```diff
@@ -1818,7 +1825,7 @@
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": "alertsmanagementresources\r\n| where type == 'microsoft.alertsmanagement/alerts'\r\n| extend severity = tostring(properties[\"essentials\"][\"severity\"])\r\n| extend startTime = properties[\"essentials\"][\"startDateTime\"]\r\n| extend description = properties[\"essentials\"][\"description\"]\r\n| extend condition = properties[\"essentials\"][\"monitorCondition\"]\r\n| where properties[\"essentials\"][\"targetResource\"] =~ '{ase}'\r\n| project [\"Alert Time\"]=todatetime(startTime),[\"Severity\"]=severity,[\"Alert\"]=name,[\"Description\"]=description,[\"Alert Condition\"]=condition\r\n| order by [\"Alert Time\"] desc",
+              "query": "alertsmanagementresources\r\n| where type == 'microsoft.alertsmanagement/alerts'\r\n| extend severity = tostring(properties[\"essentials\"][\"severity\"])\r\n| extend startTime = properties[\"essentials\"][\"startDateTime\"]\r\n| extend description = properties[\"essentials\"][\"description\"]\r\n| where properties[\"essentials\"][\"targetResource\"] =~ '{ase}'\r\n| where properties[\"essentials\"][\"monitorCondition\"] in~ ('Fired')\r\n| project [\"Alert Time\"]=todatetime(startTime),[\"Severity\"]=severity,[\"Alert\"]=name,[\"Description\"]=description\r\n| order by [\"Alert Time\"] desc",
```

Comment:
I don't think you intended to include this change - this partially reverts the fix made last week in #2406